lib/linter.js

```diff
@@ -699,12 +750,17 @@ function runRules(sourceCode, configuredRules, ruleMapper, parserOptions, parser
 
     Object.keys(configuredRules).forEach(ruleId => {
         const severity = ConfigOps.getRuleSeverity(configuredRules[ruleId]);
+        const rule = ruleMapper(ruleId);
```

Comment:
Would you move this getting below the severity check? Because it triggers to load the rule lazily (#11705). Many configs contain `"off"` settings in order to verify the config contains all rules with a tool, so this causes to load all core rules anyway.